dev/source/docs/style-guide.rst

```diff
@@ -356,6 +356,30 @@ underscores.
 
     class ap_compass { };
 
+Enums
+---------------
+
+Prefer enum classes over raw enums. Enums should be CapitalCase, and be singular.
+
+**Right:**
+
+::
+
+    enum class CompassType {
+        FOO,
+        BAR
```

Comment:
```suggestion
        BAR,
```

Do we need some words around this?  Adding hthe trailing comma simplifies the diffs when scanning down the page in github etc

Reply:
Black would definitely add the commas. Not sure about flake8.

Reply:
> Black would definitely add the commas. Not sure about flake8.

Those are both python formatters, this is a c++ style guide

Reply:
Yep. I've spent too long writing slots and enums python and c++ recently. It's all blurring together.

Reply:
I tested `astyle`, it does not care about trailing commas, so there is no way to enforce it. Sure, I'll add it to the wiki, because I agree it makes sense, but it's not something we can test for.